.github/workflows/checks.yaml

```diff
@@ -8,8 +8,8 @@ on:
   pull_request: ~
 
 env:
-  CACHE_VERSION: 27
-  DEFAULT_PYTHON: "3.10"
+  CACHE_VERSION: 29
```

Comment:
@Pierre-Sassoulas Btw, we could also make this a repository secret. It would mean that you need to update it in the settings, but it might be easier than having to do so in PRs every time.
Same goes for `DEFAULT_PYTHON` and `PRE_COMMIT_CACHE`.

Reply:
I'd rather make it accessible for everyone for now because I don't want to freeze pylint development if I'm not available. It would help if PyCQA was more democratic and I could add you to the admin group.

Reply:
Yeah, but for some reason I feel like for `pylint` this is different. We also struggle with `match ... case` statements but we still support that version according to our classifier. For `astroid` I feel like we don't really introspect the `ast` fully but for `pylint` we do offer our current functionality completely on 3.11.
The objectives are a bit different, if you understand what I mean. But I'll leave this to you to decide!

Reply:
I think you're answering the other comment about package metadata ? Let's add the classifier then.

Reply:
Ah yeah, my bad!